instrumentation/opentelemetry-instrumentation-openai-v2/tests/test_chat_completions.py

```diff
@@ -102,24 +95,16 @@ def test_chat_completion_404(
     llm_model_value = "this-model-does-not-exist"
     messages_value = [{"role": "user", "content": "Say this is a test"}]
 
-    exception = None
-    try:
+    with pytest.raises(NotFoundError):
         openai_client.chat.completions.create(
             messages=messages_value,
             model=llm_model_value,
-            timeout=0.1,
```

Comment:
I removed the timeout and added the correct expectation as at first this failed for the wrong reason (timeout)